hlc_wp_ak/config.cpp

```diff
@@ -947,14 +972,14 @@ class CfgWeapons {
 
     class hlc_rifle_ak74 : hlc_ak_base {
         dlc = "Niarms_AK";
+		author = "MrRifleman, Toadie";
+		scope = public;
         AB_barrelTwist=7.87;
         AB_barrelLength=16.3;
         ACE_barrelTwist = 199.898;
         ACE_barrelLength = 414.02;
         deployedPivot = "deploypivot";       /// what point should be used to be on surface while unfolded
-        author = "MrRifleman, Toadie";
-        scope = public;
-        displayName = "Izhmash AK74";
+		displayName = "Izhmash AK74 [16.3 inch]";

```

Comment:
Why the inclusion of barrel lengths in the Displayname?

Reply:
1. I started this in the saw addon, because it really is confusing which barrel is shorter or not, there I proposed and explained why this can be an improvement, other modders, even BI have made that their standard, at least with some weapons.
2. the barrel length can be a factor in the choice of weaponry, this distinction allows it easily.
3. In the ak addon it is especially important to understand how the barrel length will take effect with the correct ammunition. e.g.: If you want to shoot subsonic, you cannot take an 16 inch ak with a suppressor and subsonic ammunition. It will still shoot supersonic.
4. there are some suboptimal and confusional terms in the saw addon which lets one believe they have, e.g. the para version, but instead they have the same barrel length as all the long barrel,
The many saws there are cannot really be distincted in between by just the name and "long" or "short". especially not, when you want to compare it with other weapons and dont have the virtual arsenal available to gain a situational overview.
even when "long" can help to see the difference, but a number is a more representive and more accurate way to ensure the desired outcome.